contact.html

```diff
@@ -55,13 +63,20 @@
                 </div>
                 <div class="inputRow">
                     <!-- Send button -->
-                    <button type="submit">Send</button>
+                    <button class="button filled-button fakeAlert" type=submit>Send</button>
```

Comment:
We maybe shouldn't remove the quotes around "submit". Idk if it'll actually break anything according to Google, but they were there before so I'm just gonna add them back.